app/components/dsfr/combobox_component.rb

```diff
@@ -37,18 +38,20 @@ def input_class
   end
 
   def input_value
-    selected.present? ? options_with_values.find { _1.last == selected }&.first : nil
+    selected.present? ? options_with_values.find { _1.last == selected }&.first : @value
```

Comment:
```suggestion
    return @value if url.present?
    selected.present? ? options_with_values.find { _1.last == selected }&.first : nil
```
j'ai l'impression que ca explicite plus l'intention